### ISSUE TITLE
Issue while installing nextcloud Repo does not exist

### ISSUE THREAD

**Issue by feigenmann** [mtthp/hassio-addons#10]
404 Client Error for http+docker://localhost/v1.40/images/create?tag=20.0.3&fromImage=mtthp%2Faarch64-addon-nextcloud: Not Found ("pull access denied for mtthp/aarch64-addon-nextcloud, repository does not exist or may require 'docker login': denied: requested access to the resource is denied")

**Comment by mtthp** [comment]
Hello @feigenmann 👋

I'm sorry, I wasn't able to build the ARM architectures using the GitHub worker or locally. For now, the best option you have is to clone the repository and use it locally until I find a way to push the missing Docker images.

**Comment by sbonn13** [comment]
Hello @mtthp, 

Does that mean that I cannot use this addon with a Raspberry Pi?
I am trying to use your Addon since end of last year, but have had no success yet. 

Forbidden
You don't have permission to access this resource.
Apache/2.4.38 (Debian) Server at 192.168.178.013 Port 9003

**Comment by mtthp** [comment]
Hi @sbonn13,

I finally did push a Docker image for `armhf` et `armv7`. Can you try again please and let me know if it works ?

I'm struggling a little bit for `aarch64`.

**Comment by sbonn13** [comment]
Hi @mtthp ,

Oh thanks a lot! Unfortunately I cannot see the pushed image. Last commit is from Jan 3, 2021?

When I freshly install it nevertheless I receive following error, which is caused probably by the aarch64 architecture you are struggeling with (sorry have a pi4):
404 Client Error for http+docker://localhost/v1.40/images/create?tag=20.0.3&fromImage=mtthp%2Faarch64-addon-nextcloud: Not Found ("pull access denied for mtthp/aarch64-addon-nextcloud, repository does not exist or may require 'docker login': denied: requested access to the resource is denied")

**Comment by mtthp** [comment]
Yeah, that's alright. I will find a way soon enough. Thank you for trying nonetheless! 👍

**Comment by sbonn13** [comment]
Cheers buddy, looking forward to it :-)

**Comment by g189ffin** [comment]
HI just wondering if this was fixed, I'm still getting 

404 Client Error for http+docker://localhost/v1.41/images/create?tag=20.0.3&fromImage=mtthp%2Faarch64-addon-nextcloud: Not Found ("pull access denied for mtthp/aarch64-addon-nextcloud, repository does not exist or may require 'docker login': denied: requested access to the resource is denied")

Thanks

**Comment by riddertommie** [comment]
The same here, any change of this getting fixed, no is also a answer then i will have to find something else instead.

Thanks! 

404 Client Error for http+docker://localhost/v1.41/images/create?tag=20.0.3&fromImage=mtthp%2Faarch64-addon-nextcloud: Not Found ("pull access denied for mtthp/aarch64-addon-nextcloud, repository does not exist or may require 'docker login': denied: requested access to the resource is denied")

**Comment by tkupka** [comment]
Hello still no luck with `aarch64` image ?

**Comment by marekbrezina** [comment]
Morning, `aarch64 `would be very appreciated by me as well.

**Comment by Carunga** [comment]
> Morning, `aarch64 `would be very appreciated by me as well.

Yes, here tool. Anyways, great work @mtthp!

**Comment by ornago** [comment]
Just found your addon but got the same error with my RPi4 4G. Couldn't you solve the problem by now?

**Comment by DannenmannA** [comment]
Hi,
same here. Would like to install it on a RPi4, but I get the same error. Is there any possibility to contribute for the right image?
Nevertheless thank you very much 👍

**Comment by WAPEETY** [comment]
Same problem on RPI4 2gb

**Comment by cbytestech** [comment]
same on 8gb
rpi4

**Comment by cbytestech** [comment]
hey @mtthp Is there a script and process to get to where I can enter the script to install via terminal plugin instead of the one that runs by default with the plugin?

this way I can try the aarch push you put through

**Comment by bobloadmire** [comment]
did anyone find a workaround for this?

**Comment by realEntwickler** [comment]
Nope, still experiencing this error today.

**Comment by tkupka** [comment]
Use the: https://github.com/alexbelgium/hassio-addons/tree/master/nextcloud
